Source/Magick.NET/Shared/Profiles/Exif/ExifValue.cs

```diff
@@ -125,6 +125,82 @@ internal int NumberOfComponents
             }
         }
 
+        /// <summary>
+        /// Converts the string representation of a value to the correct representation based on the EXIF tag to generate.
+        /// </summary>
+        /// <param name="exifTag">The EXIF tag of the resulting EXIF value</param>
+        /// <param name="value">The value to try and set</param>
+        /// <param name="result">The newly created ExifValue</param>
+        /// <returns>Indicates whether the conversion was successful or not.</returns>
+        public static bool TryParse(ExifTag exifTag, string value, out ExifValue result)
```

Comment:
I think I rather have a non static method called TrySetValue(object value) instead and do the conversion inside that method. Maybe this needs a separate class?

Reply:
I did it this way because I'm used to using TryParse in applications and the point of this change was to allow me to try and safely convert a string value to an ExifValue before I add it to the profile. The application I'm writing allows certain values to be changed, but it's generic, so using TryParse helps in the calling application.